### DISCUSSION TITLE
GNOME Software making network requests to servers not configured as repositories

### DISCUSSION THREAD

**Discussion by c-mart** [evilsocket/opensnitch#1127] [Show and tell]
Installed OpenSnitch today, noticed this a few minutes later. The GNOME Software app wasn't even open. :slightly_smiling_face: 

https://gitlab.gnome.org/GNOME/gnome-software/-/issues/2556

**Comment by gustavo-iniguez-goya** [comment]
:O wow, that's really unexpected, but there's something weird going on there.

It's trying to establish a UDP connection to 208.91.112.53 , port 53, which seems to be a Fortinet DNS nameserver. ok.
But why would gnome-software be connecting to web.telegram.org ? Maybe we're associating the PID with the wrong application?

If you can reproduce it, copy the PID while the pop-up is active, open a terminal and see if the PID belongs to the application displayed on the pop-up (`pgrep -a <pid>`).

**Reply by c-mart** [reply]
Yes, I can reproduce it (by opening GNOME Software, or deleting the OpenSnitch deny rule and waiting a bit). The PID indeed belongs to to `/usr/bin/gnome-software --gapplication-service`.

I believe that GNOME Software is doing a DNS lookup for `web.telegram.org`, and 208.91.112.53 happens to be the DNS server IP that DHCP hands out on the wifi I was connected to. (The DNS hostname changes when I connect my computer to a different network.)

> But why would gnome-software be connecting to web.telegram.org ?

My [GNOME Software log output](https://gitlab.gnome.org/GNOME/gnome-software/uploads/dcc22356fd2341caf6ea3971b6fe63cd/gnome-software.log) suggests that it's trying to download an icon file for the Telegram software package.

**Reply by gustavo-iniguez-goya** [reply]
aah, you're right, it tries to download an icon (as you already reported): `00:05:58:389 Gs  failed to cache icon for org.gnome.Software.WebApp_78977f7aec98d1c072a768b6a366b8d99cde8df4.desktop: Error resolving “web.telegram.org”: Name or service not known`

How good to see that it's being useufl :) 
Thank you for letting us know!
